app/javascript/dashboard/components/ChatList.vue

```diff
@@ -219,7 +219,7 @@ export default {
       folders: 'customViews/getCustomViews',
     }),
     hasAppliedFilters() {
-      return this.appliedFilters.length;
+      return this.appliedFilters.length !== 0;
```

Comment:
This change, it will fix showing a warning in the console when we apply a filter, which you can see in the below screencast.


https://user-images.githubusercontent.com/64252451/155257777-2d925351-3ce3-4825-a4e3-f7814b258431.mov